google/cloud/documentai_toolbox/wrappers/hocr.py

```diff
@@ -0,0 +1,221 @@
+import dataclasses
+from typing import List, Union
+from google.cloud import documentai
+
+from google.cloud.documentai_toolbox.constants import ElementWithLayout
+
+
+def _get_bounding_box(element_with_layout: ElementWithLayout, dimensions: documentai.Document.Page.Dimension):
+    if element_with_layout.layout.bounding_poly.vertices:
```

Comment:
Looks like there are `vertices` and there are `normalized_vertices`.  What are the situations where only `normalized_vertices` are populated but not `vertices`?  (That looks like the only case where `dimensions` is used, and right not we pass this to the hOCR classes just for this purpose.  This perhaps could be avoided.)

Reply:
i've seen cases where the OCR doesn't populate vertices, but i'll test this to validate that it happens sometimes otherwise if it doesn't happen the dimensions is not needed.

Reply:
Can we also confirm this with the service team?